### PR TITLE
Lunch and Learn FAQ

### PR DESCRIPTION
**Pull request by orta**
Covers a lot of the basics, and makes it easy to look up what we do for external contributors

also /cc @ashfurrow

### CODE REVIEW

File: meta/lunch_and_learn.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+## Format
+
+The format is definitely not set in stone, however these are what has worked for us internally and for past contributors.
+
+#### Presentation
+
+Traditional conference/meetup style: You bring slides, we watch. You talk, we listen. You finish, we ask questions.
+
+We don't expect conference/meetup-level presentation, and practice. This is a great oppertunity to present new talks to new eyes.
```
Comment (ashfurrow) [inline meta/lunch_and_learn.md]: > We don't expect conference/meetup-level presentation, and practice.

Could use some clarification – a lack of polish is okay. Love the note about the opportunity for new talks, maybe say "try new talk ideas"? 

"oppertunity"

**Review by ashfurrow — APPROVED** [review]

**Comment by dblock** [comment]
Spell checker please, "sparodic"?

File: meta/lunch_and_learn.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+We don't expect conference/meetup-level presentation, and practice. This is a great oppertunity to present new talks to new eyes. 
+
+#### Company Walkthrough
+
+More informal, you can come with other speakers and talk about the ways in which your company works. Covering your tech stack, the decisions behind them. With the presentation mainly being a series of demos, and showing code architecture.
+
+## Schedule
+
+* Arrive at Artsy for 11:30am ( We're [401 Broadway, 25th Floor, 10013](https://foursquare.com/v/artsy/4f53d65de4b0b589399898a1) - Canal St is the closest station.)
```
Comment (dblock) [inline meta/lunch_and_learn.md]: We're moving the reception to the 26th floor.

Comment (dblock) [reply meta/lunch_and_learn.md]: Maybe FourSquare isn't the best link? Everybody uses Google Maps though.

File: meta/lunch_and_learn.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+* Ask for [Orta], [Craig] or [Ash]
```
Comment (dblock) [inline meta/lunch_and_learn.md]: Maybe clarify that they ask for the person they have been talking to.

File: meta/lunch_and_learn.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+* One of us will get you set up on our 27th floor Classroom
+* We will set up [the recording](#recording) and slides
```
Comment (dblock) [inline meta/lunch_and_learn.md]: Add something like "if you don't want to be recorded ..." and/or something about who this video is shared with (internally)

File: meta/lunch_and_learn.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+* There is inevitable technical difficulties (~10 minutes)
```
Comment (dblock) [inline meta/lunch_and_learn.md]: are

**Comment by orta** [comment]
Cool, agreed with all that 👍

**Pull request merged**